.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "6"
-  - "8"
+  - "12"
+  - "14"
```

Comment:
Does this mean it's a breaking change since it no longer runs the tests on node 6?

(Or is it running the tests on the source code, and the built library runs fine on the same browser/node versions?)

Reply:
Some updates to Rollup and Rollup plugins use `async` which wasn't available in Node 6, so the actual build fails.